### PR TITLE
Portofolio welcoming page

### PR DESCRIPTION
**Pull request by ishimwezachee**
In this project, I implemented the following 
 - created two-section such as header and section using HTML and CSS 
 - implemented responsiveness so that its clear in the mobile version 
 - configured linters to have well organized codes 
 - documented the project based on the given template 
 - added fontAwesome CDN for me to have access to the icons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <title>Portofolio</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navigation">
+        <h1 id="wlcm">Welcome</h1>
```
Comment (SajjadAhmad14) [inline index.html]: Please wrap the logo in an anchor tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            <a href="#">Portofolio</a>
+          </li>
+          <li>
+            <a href="#">About</a>
+          </li>
+          <li>
+            <a href="#">Contact</a>
+          </li>
+        </ul>
+        <i class="far fa-bars"></i>
```
Comment (SajjadAhmad14) [inline index.html]: Please use the `nav` tag for the hamburger menu only and not for the logo.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <title>Portofolio</title>
+  </head>
+  <body>
+    <header>
```
Comment (SajjadAhmad14) [inline index.html]: Please use color and font family for the Toolbar section from the design page.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+  font-size: 25px;
+  color: #172840;
+  margin-left: 2%;
+  margin-top: 2%;
+}
+
+@media only screen and (min-device-width: 320px) and (max-device-width: 480px) {
+  .nav-list {
+    display: none;
+  }
```
Comment (SajjadAhmad14) [inline style.css]: Please do not use media queries here. It's a mobile-first design.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,105 @@
+html {
+  height: 100%;
+}
+
+body {
+  background-image: url("./images/Header-llustration-desktop.png");
```
Comment (SajjadAhmad14) [inline style.css]: Please add background images to the Headline section instead of the body.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestate to Contact me.
+        </p>
+        <div class="icons">
+          <i class="fab fa-github-square"></i>
+          <i class="fab fa-linkedin-in"></i>
+          <i class="far fa-hand-peace"></i>
+          <i class="fab fa-twitter"></i>
+          <i class="fab fa-medium-m"></i>
+        </div>
```
Comment (SajjadAhmad14) [inline index.html]: The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag).

File: index.html [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+            <a href="#">About</a>
+          </li>
+          <li>
+            <a href="#">Contact</a>
+          </li>
+        </ul>
+        <i class="far fa-bars"></i>
+      </nav>
+    </header>
+    <section id="welcome-section">
```
Comment (SajjadAhmad14) [inline index.html:27]: Please use `flexbox` to align content in the Headline section.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <title>Portofolio</title>
+  </head>
+  <body>
```
Comment (SajjadAhmad14) [inline index.html:16]: Page is overflowing horizontally. This means that some content is overflowing from its container. Please fix the issue.
overflow

File: index.html (hunk 3 above)
Comment (SajjadAhmad14) [inline index.html]: Background image for the Headline section should position under the hamburger menu and it should not overflow on the hamburger menu. Kindly see the screenshot below and follow the design to place background images on the page.
header-back

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Zachee Ishimwe, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by mke2111 — APPROVED** [review]
### Hi @ishimwezachee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**